Events Explorer (#11323)
DataDog/documentation: content/en/events/guides/_index.md

    - /events/
---

> Datadog classifies an _event_ as any notable changes relevant for managing IT operations such as code deployments, service health, configuration changes, or monitoring alerts. Datadog Events gives you a consolidated interface to search, analyze, and filter events from any source in one place. Use these guides to progamattically send events:

Reviewer: ```suggestion
Datadog classifies an _event_ as a record of any notable change relevant to managing IT operations such as code deployments, service health, configuration changes, or monitoring alerts. Datadog Events gives you a consolidated interface to search, analyze, and filter events from any source in one place. Use these guides to programatically send events:
```

    {{< nextlink href="/api/v1/events/#post-an-event" >}}Datadog API{{< /nextlink >}}
{{< /whatsnext >}}

> Use this guide if you are migrating from the Events Stream to the Events Explorer:

Reviewer: ```suggestion
Use this guide if you are migrating from the Event Stream to the Events Explorer:
```

{{< whatsnext >}}

>     {{< nextlink href="/events/guides/migrating_from_stream_to_explorer/" >}}Migrating from the Events Stream to the Events Explorer{{< /nextlink >}}

Reviewer: ```suggestion
    {{< nextlink href="/events/guides/migrating_from_stream_to_explorer/" >}}Migrating from the Event Stream to the Events Explorer{{< /nextlink >}}
```
